### ISSUE TITLE
Crash : [UIView(PAGUgenWidget) ugenLayoutSubviews]  when using SwiftUI previews

### ISSUE THREAD

**Issue by valougep** [bytedance/AdsGlobalPackage#52]
Crash on SPM package version 5.9 already but still present on version 6.1
Reproduce when opening a swiftUI preview in an app using the package

```
Exception Type:  EXC_BAD_ACCESS (SIGSEGV)
Exception Subtype: KERN_PROTECTION_FAILURE at 0x000000016cdf3f90
Exception Codes: 0x0000000000000002, 0x000000016cdf3f90
Triggered by Thread:  0

Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   MyApp        	       0x103c9f0cc -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 4
1   MyApp        	       0x103c9f0e4 -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 28
2   MyApp        	       0x103c9f0e4 -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 28
3   MyApp        	       0x103c9f0e4 -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 28
4   MyApp        	       0x103c9f0e4 -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 28
5   MyApp        	       0x103c9f0e4 -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 28
6   MyApp        	       0x103c9f0e4 -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 28
7   MyApp        	       0x103c9f0e4 -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 28
8   MyApp        	       0x103c9f0e4 -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 28
9   MyApp        	       0x103c9f0e4 -[UIView(PAGUgenWidget) ugenLayoutSubviews] + 28
...

Thread 0 crashed with ARM Thread State (64-bit):
    x0: 0x000000010c210820   x1: 0x0000000104bddf85   x2: 0x00000001ec289768   x3: 0x000000010c307a20
    x4: 0x00000000612e19a0   x5: 0x0000000000000020   x6: 0x0000000000000000   x7: 0x0000000000000403
    x8: 0x00000001ec31a000   x9: 0x0000000104bddf85  x10: 0x000000010b832400  x11: 0x00000000000000ff
   x12: 0x0000000000000085  x13: 0x000000010b832c40  x14: 0x03000001ec2a7835  x15: 0x00000001ec2a7830
   x16: 0x00000001ec2a7830  x17: 0x0000000103c9f0c8  x18: 0x0000000000000000  x19: 0x000000010c210820
   x20: 0x0000000000000000  x21: 0x0000000000000000  x22: 0x00000000000000da  x23: 0x0000000000000000
   x24: 0x0000000000000000  x25: 0x00000001ec09de28  x26: 0x00000001ec261000  x27: 0x0000000000000000
   x28: 0x000000016d5e9de0   fp: 0x000000016cdf4020   lr: 0x0000000103c9f0e4
    sp: 0x000000016cdf3f50   pc: 0x0000000103c9f0cc cpsr: 0x60001000
   far: 0x000000016cdf3f90  esr: 0x92000047 (Data Abort) byte write Translation fault
```

**Comment by valougep** [comment]
Crash is present also on your other repos:
-  https://github.com/bytedance/Bytedance-UnionAD/issues/202
- https://github.com/bytedance/Bytedance-UnionAD/issues/206

**Comment by zry-bd** [comment]
We have fixed this issue in versions 6.2.0 and above, please update pangle SDK version.
